styles/_footer.scss

```diff
@@ -1,20 +1,195 @@
 /***** Footer *****/
-.footer {
-  border-top: 1px solid $border-color;
-  margin-top: 60px;
-  padding: 30px 0;
+footer {
+  margin: 0 auto;
+  @include fluid-value( padding-top, $min-width, $max-width, $whitespace-medium, $whitespace-medium--desktop );
+  max-width: $max-width-page-container;
+  background-color: $primary--light;
+  color: $white;
 
-  a { color: $secondary-text-color; }
+  .footer-inner {
+    display: block;
 
-  &-inner {
-    @include max-width-container;
-    display: flex;
-    justify-content: space-between;
+    &-column-wrap {
+      @include fluid-value( margin-top, $min-width, $max-width, $whitespace-small, $whitespace-small--desktop );
+      
+      @include tablet{
```

Comment:
does the linter not catch this missing space?

Reply:
It did not! Help?